libraries/provider_rack_user.rb

```diff
@@ -1,4 +1,5 @@
 require 'chef/resource/lwrp_base'
+require 'shadow'
```

Comment:
Are we relying on Chef to install the shadow gem? I don't see it installed anywhere, and they've been known to swap out gems between releases (rest_client, for instance) without giving any notice.